screenshot.py

```diff
@@ -1,11 +1,15 @@
 import os
+import platform
 
 
 def capture_screenshot(filename):
     """
     Captures a screenshot of the Android screen using adb and saves it to a file.
     Returns True if the adb command was successful, False otherwise.
     """
-    adb_command = "adb exec-out screencap -p > {} 2> /dev/null".format(filename)
+    if platform.system() == "Windows":
+        adb_command = "adb exec-out screencap -p > {}".format(filename)
+    else:
+        adb_command = "adb exec-out screencap -p > {} 2> /dev/null".format(filename)
```

Comment:
I made a change for the screenshot command, so that it checks if the user is running windows or not. If not it uses the old command which suppressed printing error messages. If the user runs windows it uses your suggestion.